packages/jest-worker/src/Farm.ts

```diff
@@ -75,23 +75,17 @@ export default class Farm {
 
         if (computeWorkerKey) {
           hash = computeWorkerKey.call(this, method, ...args);
-          worker = hash == null ? null : this._cacheKeys[hash];
+          worker = hash === null ? null : this._cacheKeys[hash];
         }
 
         const onStart: OnStart = (worker: WorkerInterface) => {
-          if (hash != null) {
+          if (hash !== null) {
```

Comment:
Note that while `===` and `!==` are general best practice,s this is changing the behaviour of this function, because previously if `hash` was undefined it was still consider null-y, but now it's not, and will end up storing the worked on cache key`"undefined"`, which is probably bug.

See:
```
null == undefined
> true
null === undefined
> false
```

This change may not be safe.

Reply:
I see this has already been addressed in the next commit here https://github.com/MLH-Fellowship/jest/pull/5/commits/9fd026e1aef025797b3e9c75e87a2750a0637414

👍

Reply:
Yeah, I've reverted the change. You are right :grimacing: